### PR TITLE
Pr 60

### PR DESCRIPTION
**Pull request by rough007**
Adding Mac and Linux collection

### CODE REVIEW

File: CyLR/src/CollectionPaths.cs [hunk 1]
```diff
@@ -28,11 +29,81 @@ public static List<string> GetPaths(Arguments arguments, List<string> additional
...
+                        UseShellExecute = false,
+                        RedirectStandardOutput = true,
+                        CreateNoWindow = true
+                    }
+                };
+                proc_shellhistory.Start();
+                while (!proc_shellhistory.StandardOutput.EndOfStream)
+                {
+                    string line = proc_shellhistory.StandardOutput.ReadLine();
+                    defaultPaths.Add(line);
```
Comment (Lansatac) [inline CyLR/src/CollectionPaths.cs]: It might be good to pull these out into functions. Since the `find` commands only vary by argument, we could just pass that into a function and then that'll collapse this nicely.

File: CyLR/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -32,7 +32,7 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.3.3.0")]
-[assembly: AssemblyFileVersion("1.3.3.0")]
+[assembly: AssemblyVersion("1.3.4.0")]
+[assembly: AssemblyFileVersion("1.3.4.0")]
```
Comment (Lansatac) [inline CyLR/Properties/AssemblyInfo.cs]: It doesn't matter for now, but as a convention, it's usually best to do version changes in a separate commit.

**Comment by rough007** [comment]
Code updated
